pkg/test/framework/components/echo/annotations.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package echo
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type AnnotationType string
```

Comment:
`pkg/annotations/annotation.go` already has some shared lib for anntation. should we reuse? and extend if not meets our requirements?

Reply:
The annotations package doesn't have the concept of default values and it doesn't need to.  The annotations here need defaults since we have logic that depends on it (sidecar/no-sidecar).

I don't think it's worth while at this time to attempt to merge the requirements of both.